tests/validation_suite/results/worker_lookup_request.json

```diff
@@ -0,0 +1 @@
+###
```

Comment:
Empty file?

Reply:
Replaced with proper json request file, created for debugging purpose.